deploy_nixos/nixos-instantiate.sh

```diff
@@ -32,7 +32,7 @@ fi
 cd "$(readlink -f "$config_pwd")"
 
 # Run!
-echo "running: ${command[*]@Q}" >&2
+echo "running: $(env | grep NIX_PATH) ${command[*]@Q}" >&2
```

Comment:
```suggestion
echo "running: ${NIX_PATH:+NIX_PATH=$NIX_PATH} ${command[*]@Q}" >&2
```
`env | grep` might return more than one result.